### ISSUE TITLE
Redo list commmand and change api/resources command to use Json format

### ISSUE THREAD

**Issue by Pythonic-Rainbow** [RbxAPI/Docs-Bot#23]
Sample of new list command:
![image](https://user-images.githubusercontent.com/25037295/72207545-188d3b80-3492-11ea-9ac8-7234a136517d.png)

Proposed new api_list.json format:
```json
{
    "RandomLanguage": {
        "RandomLibrary": {"https://github.com/RandomRepo", randomAuthorGithub(randomAuthorDiscord), randomChannel},
    }
}
```

And also creating new api.json and resource.json instead of using current method
```python
embed.add_field(name, value)
```
that hard codes fields

**Comment by Pythonic-Rainbow** [comment]
If we are creating the json files, might create a json folder and **copy** current `api_list.json` to it.
The reason to copy but not move is Sanjay using Dynamic Update for his bot
i.e. let the file stay until Sanjay polls the change to his main bot, which uses the new directory

**Comment by sbhadr** [comment]
Perhaps. Maybe it's better to house all ``.json`` files in a separate directory to keep things organized.

**Comment by Pythonic-Rainbow** [comment]
so it will be like
```
/json/api_list.json
/json/resources.json
/json/api.json
/main.py
...
```
Might as well rename api_list to another file name to prevent confusion

**Comment by sbhadr** [comment]
I've pushed a commit upstream that corrects the issue earlier with ``Auto.py`` and have removed it completely. As a result, I've made the method call the git repo per invoke rather than just once like previous. https://github.com/RbxAPI/Docs-Bot/commit/8cfb33359a79fca1c14302b316540ca6f6e74e16

https://github.com/RbxAPI/Docs-Bot/blob/rewrite/main.py#L27

As for the directory changes, I agree that it should be in its own directory like you've stated.
